cmd/oras/internal/option/packer.go

```diff
@@ -91,27 +91,24 @@ func (opts *Packer) Parse(*cobra.Command) error {
 			return fmt.Errorf("%w: %v", errPathValidation, strings.Join(failedPaths, ", "))
 		}
 	}
-	return nil
+	return opts.parseAnnotations(cmd)
 }
 
-// LoadManifestAnnotations loads the manifest annotation map.
-func (opts *Packer) LoadManifestAnnotations() (annotations map[string]map[string]string, err error) {
-	if opts.AnnotationFilePath != "" && len(opts.ManifestAnnotations) != 0 {
-		return nil, errAnnotationConflict
+// parseAnnotations loads the manifest annotation map.
+func (opts *Packer) parseAnnotations(cmd *cobra.Command) (err error) {
+	if opts.AnnotationFilePath != "" && len(opts.ManifestAnnotationFlags) != 0 {
+		return errAnnotationConflict
 	}
 	if opts.AnnotationFilePath != "" {
-		if err = decodeJSON(opts.AnnotationFilePath, &annotations); err != nil {
-			return nil, &oerrors.Error{
+		if err = decodeJSON(opts.AnnotationFilePath, &opts.Annotations); err != nil {
```

Comment:
`opts.Annotations` set by `opts.AnnotationFilePath` will be overwritten by `opts.ManifestAnnotationFlags` as per `annotation.go:45`.

Reply:
Those flags cannot be used together.